docs/shared-content/octopus-recommendations/project-recommendations.include.md

```diff
@@ -53,7 +92,7 @@ These conditions allow you to have a greater degree of control over your deploym
 
 ## Automate every component's deployment {#automate-every-components-deployment}
 
-A typical scenario we see is that application deployments are automated, but the database piece is not.  That's still a manual process that means a DBA has to run the scripts on the night of deployment to production.  After they finish, the automated process can be kicked off.  Because this is manual, there's a good chance one or more of the scripts were not included in the deployments to dev, testing or staging.  Without prior testing, the likelihood of success goes down, and the deployment time goes up.
+A typical scenario we see is that application deployments are automated, but the database piece is not.  That's still a manual process which means a DBA has to run the scripts on the night of deployment to production.  After they finish, the automated process can be kicked off.  Because this is manual, there's a good chance one or more of the scripts were not included in the deployments to dev, testing or staging.  Without prior testing, the likelihood of success goes down, and the deployment time goes up.
```

Comment:
```suggestion
A typical scenario we see is that application deployments are automated, but the database deployment is manual.  This means a DBA must run the scripts on the night of deployment to production.  After they finish, the automated process can be kicked off.  Because this is manual, there's a good chance one or more of the scripts were not included in the deployments to dev, testing or staging.  Without prior testing, the likelihood of success decreases and the deployment time increases.
```